char-sheet/src/components/EquipmentTable.tsx

```diff
@@ -60,7 +61,25 @@ function EquipmentTable({equipment, container, service, editable=EditMode.Live}:
           items.map((item, i) => {
             return <tr className='hover tooltip tooltip-left w-full' data-tip={lookupDescription(item.name)} key={i}>
               <td className='w-full text-left'>
-                { item.name }
+                {item.custom ? (
+                  editingIndex === i ? (
+                    <input
+                      type='text'
+                      className='input input-bordered w-full'
+                      value={item.name}
+                      onChange={(evt) => {
+                        service.set_index(i, { ...item, name: evt.target.value });
+                      }}
+                      onBlur={() => setEditingIndex(null)}
+                      disabled={!(editable >= EditMode.Live)}
```

Comment:
Ok, so the disabled logic doesn't work super well. Try opening a sheet in read only mode. You shouldn't be able to `setEditingIndex` if `editable < EditMode.Live`.

Ie, `http://localhost:3000/?sheet=076f362a-0362-40d4-964b-dd4df23fa404`